yang/frr-vrf.yang

```diff
@@ -3,6 +3,10 @@ module frr-vrf {
   namespace "http://frrouting.org/yang/vrf";
   prefix frr-vrf;
 
+  import ietf-yang-types {
+    prefix yang;
```

Comment:
this seems unnecessary?

Reply:
@rwestphal  I added this to see if module load errors can be resolved. I will remove it while handling other review comments.